downloads/1.2.x-release-notes/1.2.27.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 1.2.27
+permalink: /downloads/1.2.x-release-notes/1.2.27/
+active: 1.2.27
+redirect_from:
+    - /downloads/1.2.x-release-notes/
+---
+
+### Overview of jBallerina 1.2.27
+
+The jBallerina 1.2.27 patch release improves upon the 1.2.26 release by addressing an [issue](https://github.com/ballerina-platform/ballerina-lang/issues/35404).
+
+You can use the update tool to update to jBallerina 1.2.27 as follows.
+
+**For existing users:**
+If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina 1.2.27 by executing the following command:
+
+> $ ballerina dist update
```

Comment:
@praneesha seems like from `1.2.14` onwards its support `bal dist update`

Reply:
Updated in https://github.com/ballerina-platform/ballerina-dev-website/pull/4071.